spec/services/nomis/api_spec.rb

```diff
@@ -185,4 +185,41 @@
       expect(subject.first.id).to eq(first_contact.id)
     end
   end
+
+  describe 'book_visit' do
+    let(:params) do
+      {
+        lead_contact: 12_588,
+        other_contacts: [13_428],
+        slot: '2017-05-15T10:00/16:00',
+        override_restrictions: false
+      }
+    end
+
+    let(:offender_id) { 1_057_307 }
+
+    subject { super().book_visit(offender_id: offender_id, params: params) }
+
+    context 'happy path', vcr: { cassette_name: 'book_visit_happy_path' } do
+      it 'returns the visit_id' do
+        expect(subject.visit_id).to eq(5_467)
+      end
+
+      it 'instruments the outcome of the call' do
+        subject
+        expect(PVB::Instrumentation.custom_log_items[:book_to_nomis_success]).to eq(true)
```

Comment:
I think it is clearer as you directly see what method call is being tested with which arguments and also remove this odd call to subject on the first line of the test:

```ruby
expect {
  subject.book_visit(offender_id: offender_id, params: params)
}.to change { PVB::Instrumentation.custom_log_items[:book_to_nomis_success] }.from(nil).to(true)